pkgs/development/php-packages/snuffleupagus/default.nix

```diff
@@ -42,10 +41,10 @@ buildPecl rec {
     ./configure --enable-snuffleupagus
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Security module for php7 and php8 - Killing bugclasses and virtual-patching the rest!";
-    license = licenses.lgpl3Only;
     homepage = "https://github.com/jvoisin/snuffleupagus";
-    maintainers = teams.php.members ++ [ maintainers.zupo ];
+    license = lib.licenses.lgpl3Only;
+    maintainers = lib.teams.php.members;
```

Comment:
Why remove zupo?

Reply:
Because I had the feeling that he was no more active: https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+author%3Azupo+is%3Aclosed

Reply:
They approved the previous PR. Also https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+involves%3Azupo+is%3Aclosed+sort%3Aupdated-desc+
Either way, the polite thing is to ask.

Reply:
I'll revert this change immediately.

Reply:
https://github.com/NixOS/nixpkgs/pull/242945

Reply:
Job done.